lib/data_hygiene/bulk_organisation_updater.rb

```diff
@@ -0,0 +1,68 @@
+require "csv"
+
+module DataHygiene
+  class BulkOrganisationUpdater
+    def initialize(filename)
+      @filename = filename
+      @all_organisations = Queries::GetLinkables.new(document_type: "organisation").call
+    end
+
+    def call
+      CSV.foreach(filename, liberal_parsing: true, headers: :first_row) do |row|
+        process_row(row)
+      end
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+  private
+
+    attr_reader :filename
+
+    def process_row(row)
+      document = find_document(row)
+      return unless document
+
+      organisations = find_organisations(row)
+      update_document(document, organisations)
+
+      puts "Sucessfully updated"
+    end
+
+    def find_document(row)
+      path = row.fetch("Path")&.strip
+      edition = Edition.find_by(base_path: path)
+
+      if edition.nil?
+        puts "error: #{path}: could not find edition"
+      else
+        edition.document.content_id
+      end
+    end
+
+    def find_organisations(row)
+      organisations = row.fetch("All organisations")&.strip
+      return [] unless organisations
+
+      new_organisations = []
+      organisations.split(",").map do |slug|
+        organisation = @all_organisations.detect { |org| org.base_path.strip == "/government/organisations/#{slug}".strip }
```

Comment:
I think we're using `strip` in the wrong places right now. We need to strip the `slug` variable, and nothing else.

I've also switched the use of `map` for `each` since we're not concerned with the output value. Instead, we're appending to the `new_organisations` array, so `each` is fine here.

Sorry to be pernickety 🙈 but how about this?

```suggestion
      organisations.split(",").map(&:strip).each do |slug|
        organisation = @all_organisations.detect { |org| org.base_path == "/government/organisations/#{slug}" }
```